image/templates/helm/shared/templates/_certrefresh.tpl

```diff
@@ -0,0 +1,19 @@
+{{/*
+  srox.setSecuredClusterCertRefresh
+
+  Checks if Secured Cluster certificate refresh is enabled in the current context, and sets
+  the securedClusterCertRefresh accordingly.
+
+  It should be enabled for Helm and Operator installs, and only on Secured Clusters.
+   */}}
+
+{{ define "srox.setSecuredClusterCertRefresh" }}
+
+{{ $ := index . 0 }}
+{{ $env := $._rox.env }}
+
+{{ if and (not $env.centralServices) (ne $env.installMethod "manifest") }}
+  {{ $_ := set $._rox "_securedClusterCertRefresh" true }}
```

Comment:
nit: Maybe set it to false otherwise to have properly initialized variables?

Reply:
Done in 34fe12b33b7428b1b1d1846d64161177e1547828